lib/plugins/features/movies/controller.js

```diff
@@ -3,6 +3,11 @@
 const Movie = require('../../../models/movie');
 
 exports.create = (payload) => {
+  if (payload.hasOwnProperty('title')) {
+    payload.name = payload.title;
+    Reflect.deleteProperty(payload, 'title');
+  }
```

Comment:
If we're not specifically trying to accommodate for both `title` and `name`, why don't we update the validators to only accept name? This part is probably useful as an intermediate step where we would be saving to both columns before we phase out `title`. 

So we *could* split this PR into two PRs where one has migrations that add the additional column (and accept either as input) and then another PR removes the `title` column. But since we're dropping the `title` column in this PR,  let's go ahead and update the validators to only use `name`?

Reply:
@sjl2 mentioned this in my other PR. See my comment [here](https://github.com/marklee22/sfmovies/pull/9#discussion_r139566333).